### ISSUE TITLE
Inadvertent stripMargin pre-processing of code listing

### ISSUE THREAD

**Issue by olafurpg** [tpolecat/tut#235]
Code fences with `tut:passthrough` are currently pre-processed by tut with `stripMargin`. Minimal reproduction
````diff
- obtained with tut 0.6.7
+ expected output
$ cat in/a.md
```tut:passthrough
println(
s"""|multiline
    |// this pipe character will disapppear"""
)
```
$ coursier launch -r "https://dl.bintray.com/tpolecat/maven/" org.tpolecat:tut-core_2.12:0.6.7 -- in out '.*\.md$' -classpath $(coursier fetch -p org.scala-lang:scala-library:2.12.6)
[tut] compiling: in/a.md
$ cat out/a.md

|multiline
-// this pipe character will disapppear
+  |// this pipe character will disapppear

````

We are currently migrating the Scalafmt website https://github.com/scalameta/scalafmt/pull/1199#issuecomment-410532132 to use tut and this bug is causing problems when rending code examples with multiline string literals such as https://scalameta.org/scalafmt/#assumeStandardLibraryStripMargin

I believe the culprit is this line here https://github.com/tpolecat/tut/blob/4caa6fc1849f99e4e91ed01bb0ce1053827ed6c2/modules/core/src/main/scala/tut/Tut.scala#L88 It's the only occurence of "stripMargin" in the codebase.

I propose to refactor that line as follows 
```scala
// before
        s"""$text
           |{: $decorations }""".stripMargin
// after
new StringBuilder()
  .append(text)
  .append("\n{: ")
  .append(decorations)
  .append(" }")
  .toString()
```
I'm happy to send a PR if the proposed solution is acceptable.

**Comment by olafurpg** [comment]
This appears to be a bug in the Scala REPL!

```
Welcome to Scala 2.12.6 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_102).
Type in expressions for evaluation. Or try :help.

scala> :paste
// Entering paste mode (ctrl-D to finish)

  println("""
  | Hello
    |World!
  """)


// Exiting paste mode, now interpreting.


 Hello
World!

```

Fortunately, it's been fixed in the latest 2.13 

```
Welcome to Scala 2.13.0-pre-693c701 (Java HotSpot(TM) 64-Bit Server VM, Java 1.8.0_102).
Type in expressions for evaluation. Or try :help.

scala> :paste
// Entering paste mode (ctrl-D to finish)

  println("""
  | Hello
    |World!
  """)

// Exiting paste mode, now interpreting.


  | Hello
    |World!

```

Guess we'll work around it on our side then.

**Comment by tpolecat** [comment]
Huh, weird. Oooookay ;-)
